adafruit_lis3dh.py

```diff
@@ -163,14 +182,14 @@ def range(self):
         return (ctl4 >> 4) & 0x03
 
     @range.setter
-    def range(self, range_value):
+    def range(self, range_value: Literal[0b00, 0b01, 0b10, 0b11]):
```

Comment:
I would use the global variables instead of individual `int` values, since after a change I make soon they'll display in RTD as that instead of the values.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7